warehouse/query-core/src/main/java/datawave/query/attributes/Attribute.java

```diff
@@ -31,6 +33,7 @@
      */
     protected Key metadata = null;
     protected boolean toKeep = true; // a flag denoting whether this attribute is to be kept in the returned results (transient or not)
+    protected boolean fromIndex; // Assume attributes are from the index unless specified otherwise.
```

Comment:
If this comment is correct, then should it be initialized to true?

Reply:
It should be. Fixed.